llvm/tools/llvm-snippy/README.md

```diff
@@ -243,8 +260,44 @@ You can reproduce the same generation by repeating the call with the same seed.
 
 Of course, not everything is covered in this quick start. We hope to publish detailed documentation soon. For now, the help command is available.
 
-# Contacts
+# Contributing to LLVM-snippy
+
+Thank you for showing interest in contributing to LLVM-snippy. These are several ways you can help to make the generator better.
+
+## Bug Reports
 
-For questions regarding llvm-snippy and its development, as well as your suggestions, please contact konstantin.vladimirov@syntacore.com
+We want to know about all LLVM-snippy bugs: segmentation faults, poor or incorrect diagnostics, lack of randomization in the generated test, incorrectly working feature, etc. If you think that you have encountered a bug, please file an issue on GitHub. Make sure that you provide minimal description, reproduction and HEAD commit hash in the issue.
+
+## Bug Fixes
+
+We appreciate your desire to improve LLVM-snippy. You can start working on any open unassigned issue. A comment in the issue is enough to show that you have started working on it. If you do not have bandwidth to work on the issue already assigned to you, please let others know by leaving a comment in the issue, so it can be reassigned.
+If the fix requires design discussion please create an [RFC](#driving-a-major-feature). Anyway, we encourage everyone to provide a short summary in the issue comments before creating a PR.
+
+## Driving a Major Feature
+
+If you want to introduce a major change or implement a new feature in LLVM-snippy, please create an RFC first. We would like to keep LLVM-snippy community informed about major changes. We should also reach consensus on all technical and design decisions before any significant work is done.
+There is no template for an RFC, feel free to create it on your own. Though usually a good RFC contains: overview, proposal, pros and cons, implementation steps.
+
+## Backward compatibility
+
+We try to keep backward compatibility in snippy for our customers. Backward incompatible changes includes, but not limited to removal of an existing option, change of input configuration format, amends in the signatures of entry functions or globals. These changes require major release version change and must be disscussed in prior. Please, create an [RFC](#driving-a-major-feature) for such changes.
```

Comment:
```suggestion
We try to keep backward compatibility in snippy for our customers. Backward incompatible changes include, but are not limited to, removing an existing option, changing an input configuration format, amends in the signatures of entry functions or globals. These changes require a major release version change and must be disscussed in prior. Please, create an [RFC](#driving-a-major-feature) for such changes.
```

Reply:
Also, do we call the users "customers"? I would change it to users or smth like this.